tests/devices/unit_tests/test_oav.py

```diff
@@ -99,3 +105,28 @@ def test_bottom_right_from_top_left():
         top_left, 15, 20, 0.005, 0.007, 1, 1
     )
     assert bottom_right[0] == 198 and bottom_right[1] == 263
+
+
+def test_when_zoom_1_then_flat_field_applied(fake_oav: OAV):
+    RE = RunEngine()
+    RE(bps.abs_set(fake_oav.zoom_controller, "1.0x"))
+    assert fake_oav.mxsc.input_plugin.get() == "PROC"
+    assert fake_oav.snapshot.input_plugin.get() == "PROC"
+
+
+def test_when_zoom_not_1_then_flat_field_removed(fake_oav: OAV):
+    RE = RunEngine()
+    RE(bps.abs_set(fake_oav.zoom_controller, "10.0x"))
+    assert fake_oav.mxsc.input_plugin.get() == "CAM"
+    assert fake_oav.snapshot.input_plugin.get() == "CAM"
+
+
+def test_when_zoom_is_externally_changed_to_1_then_flat_field_not_changed(
```

Comment:
Should: Comment to explain why this is a requirement

Reply:
The comment makes it more clear, thanks.